docs/CONTRIBUTING.md

```diff
@@ -76,8 +76,7 @@ need the following setup:
 Please note that you should not use a code auto-formatter in this codebase. Please read the
 following documents to learn more about reasons behind this decision and the recommended code style
 guide. Be sure to carefully read the
-[Rust style guide 1](https://github.com/luna/ide/blob/main/docs/style-guide.md)
-and the [Rust style guide 2](https://github.com/luna/enso/blob/main/doc/style-guide/rust.md)
```

Comment:
We shouldn't remove reference to the other document without having them unified.
Currently they both contain partially separate and complementing, partially contradictory guides.

Reply:
The link is dead, so I thought that the rust style guide was removed from enso repository. But I'll fix that anyway.